package.json

```diff
@@ -35,6 +35,8 @@
     "publish:studio": "yarn changeset publish --tag studio",
     "version:latest": "yarn changeset version && yarn generateVersions && yarn angular build",
     "publish:latest": "yarn changeset publish",
+    "version:hotfix": "yarn changeset && yarn bumpVersions && yarn build",
```

Comment:
what is the purpose of doing a full `yarn build` here? Thinks only `yarn angular build` is required?